AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -41,6 +41,7 @@ video.tv-tokyo.co.jp##.adApp-wrapper
 kurashiru.com##.PcToAppBanner
 kurashiru.com##.footer-image-wrapper
 intellipaat.com##.sticky-whatsapp-wrapper 
+girlschannel.net##a[href^="https://play.google.com"] > img[width="335"]
```

Comment:
I don't see that.
![image](https://user-images.githubusercontent.com/8361299/151228017-6632c980-da2f-40b3-b373-0ab22fbd4c89.png)

Reply:
I found it on page from issue: https://girlschannel.net/topics/3768007/

Banner on your screenshot wasn't block, because on my mind it's on footer

<details><summary>Screenshot:</summary>

<img width="1317" alt="Снимок экрана 2022-01-27 в 12 34 18" src="https://user-images.githubusercontent.com/91964807/151331812-d3729176-1163-4556-8681-009e8d8dfe9c.png">

</details><br/>

Reply:
You rule blocks Android app only
![image](https://user-images.githubusercontent.com/8361299/151423929-0822ff70-b7fc-4f0f-8f9b-15a060607257.png)